index.js

```diff
@@ -250,7 +250,7 @@ class Fuse extends Nanoresource {
 
   static unmount (mnt, cb) {
     mnt = JSON.stringify(mnt)
-    const cmd = IS_OSX ? `diskutil umount ${mnt}` : `fusermount -uz ${mnt}`
+    const cmd = IS_OSX ? `diskutil unmount force ${mnt}` : `fusermount -uz ${mnt}`
```

Comment:
Is the left or right an embarassing typo in regards to unmount/umount ? 😁

Reply:
Either works but `umount` isn't even a word! Does make the module a bit bigger though...